general/overlay/etc/network/interfaces.d/wlan0

```diff
@@ -1,4 +1,4 @@
-#auto wlan0
+auto wlan0
```

Comment:
This needs to be changed back, having the interface active by default will cause compatibility issues with other devices.